Configure

```diff
@@ -5356,6 +5357,21 @@ none) libs=' ';;
 *) libs="$ans";;
 esac
 
+: see if OpenSSL is available
+ossl_p_c='/home/linuxbrew/.linuxbrew/opt/openssl'
```

Comment:
Shouldn't this start with `$libpth`/`$libspath`

Reply:
That's a good question. I don't really have the data to know the answer for sure, having just lifted the candidate locations straight from the Net::SSLeay Makefile.PL. But keeping it simple and using infrastructure we already have are good things to do, so maybe I'll switch to iterating through `$libpth`.  If it misses something weird, people can always add hints or make adjustments as needed later.

Reply:
Now starts from $libpth.